client/transactions/declarations.d.ts

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line wpcalypso/import-docblock
+import React from 'react';
+
+declare module '@woocommerce/explat' {
```

Comment:
Is not getting recognized, why ?!

Reply:
Just had to fix the module declaration a bit 👍